x-pack/plugins/canvas/tasks/helpers/webpack.plugins.js

```diff
@@ -33,6 +37,7 @@ module.exports = {
 
   resolve: {
     extensions: ['.js', '.json'],
+    mainFields: ['browser', 'main'],
```

Comment:
This is new to me, what's it do?

Reply:
It uses the `browser` or `main` fields from package.json files to determine which code to load when importing a directory that has a package.json file (in that order). This is used by modules like `@kbn/i18n` to point to the browser version of their module, and by `@elastic/eui` to point to the pre-transpiled version of their code.